solidity/contracts/TokenStaking.sol

```diff
@@ -253,8 +249,22 @@ contract TokenStaking is Authorizations, StakeDelegatable {
             require(operators[_operator].owner == _from, "Not the same owner");
         }
 
-        topUps.initiate(_value, _operator);
-        emit TopUpInitiated(_operator, _value);
+        if (!_isInitialized(operatorParams)) {
+            // If the stake is not yet initialized, we add tokens immediatelly
+            // but we also reset stake initialization time counter.
+            uint256 newAmount = operatorParams.getAmount().add(_value);
+            operators[_operator].packedParams = operatorParams.setAmountAndCreationTimestamp(
+                newAmount,
+                block.timestamp
+            );
+            emit TopUpCommitted(_operator, newAmount);
```

Comment:
`TopUpCompleted` perhaps?

Reply:
Yep, sounds good!

Reply:
Done in https://github.com/keep-network/keep-core/pull/1929/commits/77c1fdd3f9f11fe28836855005b9f805ea298421

Reply:
So maybe for consistency we should also rename `commitTopUp` -> `completeTopUp`?

Reply:
I am not sure about it. At first sight, it makes sense, but it also feels to me like `commitTopUp` is something that I have to do (what's happening here), while `completeTopUp` sounds weird because I'd expect top-up is completed automatically after the initialization period. 🤔

Reply:
Ok, it's your call.